sapphire/simulations/groundparticles.py

```diff
@@ -902,7 +900,7 @@ class GroundParticlesSimulationWithoutErrors(ErrorlessSimulation,
     pass
 
 
-class MultipleGroundParticlesSimulation(GroundParticlesSimulation):
+class MultipleGroundParticlesSimulation(GroundParticlesGEANT4Simulation):
```

Comment:
You should create a new version of the `Multiple...` class specifically for the GEANT4 version, keeping the existing version as-is.